src/Umbraco.Core/PropertyEditors/DataEditor.cs

```diff
@@ -19,6 +19,7 @@ namespace Umbraco.Core.PropertyEditors
     public class DataEditor : IDataEditor
     {
         private IDictionary<string, object> _defaultConfiguration;
+        private IDataValueEditor _nonConfigured;
```

Comment:
I don't understand the variable name, why not just `_valueEditor` or `_dataValueEditor`?